<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1201</width>
     <height>801</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#widget{
background-color:qlineargradient(spread:pad, x1:0.275626, y1:0.715909, x2:1, y2:0, stop:0.0689655 rgba(143, 149, 225, 255), stop:1 rgba(255, 255, 255, 255));}</string>
   </property>
   <widget class="QLabel" name="loginMenuLabel">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>110</y>
      <width>341</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 36pt &quot;.AppleSystemUIFont&quot;; color:rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string>Create a New Recipe!</string>
    </property>
   </widget>
   <widget class="QLabel" name="directions">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>160</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;.AppleSystemUIFont&quot;;color:rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string>Enter recipe information</string>
    </property>
   </widget>
   <widget class="QPushButton" name="createRecp">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>580</y>
      <width>271</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:20px;
background-color: rgb(255, 160, 74);
font:rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Create Recipe</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="dNameField">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>210</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="servingField">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>260</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="dNameLabel">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>210</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>*Dish name:</string>
    </property>
   </widget>
   <widget class="QLabel" name="ssLabel">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>260</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>*Serving Size:</string>
    </property>
   </widget>
   <widget class="QLabel" name="errorLabel">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>20</y>
      <width>551</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Times New Roman&quot;; color: rgb(255, 0, 16)</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="totalCfield">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>310</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="tcLabel">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>310</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Total cost:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ttcField">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>360</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="dirField">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>410</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="notesField">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>460</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="drField">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>510</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="ttcLabel">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>360</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Time to cook:</string>
    </property>
   </widget>
   <widget class="QLabel" name="dirLabel">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>410</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>*Directions:</string>
    </property>
   </widget>
   <widget class="QLabel" name="notesLabel">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>460</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Notes:</string>
    </property>
   </widget>
   <widget class="QLabel" name="drLabel">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>510</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Dietary Restrictions:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="back">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>550</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>*required fields</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
